README.md

```diff
@@ -77,6 +79,103 @@ spec:
         value: some-value
 ```
 
+### EKS Fargate sample spec
```

Comment:
Can we move this section to where the eks-managed spec is (314), might also be good to rename the title accordingly to `### EKS Fargate (alpha-1)`